Makefile

```diff
@@ -5,6 +5,9 @@ DOCKERBUILD := docker build --build-arg http_proxy=$$http_proxy --build-arg http
 
 PIP?=pip
 
+VENV_NAME:=.venv$(VENV_PY_VERSION)
```

Comment:
I have tested pyenv a little. 
It is very handy and has plugin for virtualenv management (create, delete, activate).
How shall we install dependencies to virtualenv? Something like requirements.txt ?
So buildbot dev environment setup looks like:
```
pyenv install <some python version>
pyenv virtualenv <some python version> <venv name>
pyenv activate <venv name>
pip install -r requirements.txt

```
Is it right?

It seems that some system packages should be installed. 
For example, pip from python installed (from sources) via pyenv couldn't install twisted because 
that python was built without bz2 support. After I have installed libbz2-dev and reinstalled python
 in pyenv it worked as should.  
It is described here: https://github.com/pyenv/pyenv/wiki/Common-build-problems

Maybe it would be good to use requirements.txt in Makefile too. So both methods can be
used and there will be one packages list for both methods until pyenv tested well in
buildbot development.